docs/user-guide/filesystem.md

```diff
@@ -0,0 +1,175 @@
+# PyScript and Filesystems
+
+As you know, the filesystem is where you store files. For Python to work there
+needs to be a filesystem in which Python packages, modules and data for your
+apps can be found. When you `import` a library, or when you `open` a file, it
+is on the filesystem that Python looks.
```

Comment:
I would add **virtual** in here, as *VFS* or *virtual filesystem* is all we talk about all time